src/index.js

```diff
@@ -1,8 +1,16 @@
-import serverlessExpress from "@vendia/serverless-express";
 import { handler as internalHandler } from "HANDLER";
 import polka from "polka";
+import { ServerlessAdapter } from '@h4ad/serverless-adapter';
+import { AwsStreamHandler } from '@h4ad/serverless-adapter/handlers/aws';
+import { ApiGatewayV1Adapter } from '@h4ad/serverless-adapter/adapters/aws';
+import { DummyResolver } from '@h4ad/serverless-adapter/resolvers/dummy';
+import { PolkaFramework } from '@h4ad/serverless-adapter/frameworks/polka';
 
 const app = polka().use(internalHandler);
 
-// @ts-ignore-error
-export const handler = serverlessExpress({ app });
+export const handler = ServerlessAdapter.new(app)
+  .setFramework(new PolkaFramework())
+  .setHandler(new AwsStreamHandler({ callbackWaitsForEmptyEventLoop: false }))
+  .setResolver(new DummyResolver())
+  .addAdapter(new ApiGatewayV1Adapter())
```

Comment:
The adapter should be [ApiGatewayV2Adapter](https://serverless-adapter.viniciusl.com.br/docs/main/handlers/aws#aws-lambda-response-streaming)